### PR TITLE
[release/v2.25]Add changelog for KKP patch release 2.25.4

### PR DESCRIPTION
**Pull request by archups**
**What this PR does / why we need it**:
This PR is to add changelog for KKP patch release 2.25.4.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Comment by xrstf** [comment]
ghcl has not been updated yet, so please remove the anchor/link on the version heading and turn it into a standalone line, like https://github.com/kubermatic/kubermatic/commit/85ba80669e7b5ff850552d9347cb4e85b7e1d77c#diff-3835bce42d9bf552c483745fd5b411b68559cb8ecd10d40a122d61a95bcde0fc shows.

File: docs/changelogs/CHANGELOG-2.25.md [hunk 1]
```diff
@@ -4,6 +4,21 @@
...
+
+### Bugfixes
+
+- Fix #13393 where externally deployed Velero CRDs are removed automatically from user user cluster ([#13396](https://github.com/kubermatic/kubermatic/pull/13396))
+- Fix a bug where unrequired  `cloud-config` secret was being propagated to the user clusters ([#13366](https://github.com/kubermatic/kubermatic/pull/13366))     
+- Fix null pointer exception that occurred while our controllers checked whether the CSI addon is in use or not ([#13369](https://github.com/kubermatic/kubermatic/pull/13369))
+
+### Updates
+
+- Update OSM to v1.5.2; fixing cloud-init bootstraping issues on Ubuntu 22.04 on Azure ([#13380](https://github.com/kubermatic/kubermatic/pull/13380))
```
Comment (embik) [inline docs/changelogs/CHANGELOG-2.25.md]:
```suggestion
- Update OSM to v1.5.2; fixing cloud-init bootstrapping issues on Ubuntu 22.04 on Azure ([#13380](https://github.com/kubermatic/kubermatic/pull/13380))
```

(fixes the verify job failing at spellcheck).

**Review by embik — CHANGES_REQUESTED** [review]

**Comment by archups** [comment]
> ghcl has not been updated yet, so please remove the anchor/link on the version heading and turn it into a standalone line, like [85ba806#diff-3835bce42d9bf552c483745fd5b411b68559cb8ecd10d40a122d61a95bcde0fc](https://github.com/kubermatic/kubermatic/commit/85ba80669e7b5ff850552d9347cb4e85b7e1d77c#diff-3835bce42d9bf552c483745fd5b411b68559cb8ecd10d40a122d61a95bcde0fc) shows.

@xrstf updated 2.25 Changelog to remove anchor/link.

File: docs/changelogs/CHANGELOG-2.25.md [hunk 2]
```diff
@@ -4,8 +4,27 @@
...
 
-## [v2.25.3](https://github.com/kubermatic/kubermatic/releases/tag/v2.25.3)
+
+## v2.25.4
+
+**GitHub release: [v2.25.4](https://github.com/kubermatic/kubermatic/releases/tag/v2.25.4)**
+
+### Bugfixes
+
+- Fix #13393 where externally deployed Velero CRDs are removed automatically from user user cluster ([#13396](https://github.com/kubermatic/kubermatic/pull/13396))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.25.md]: Please turn this into a link, outside of GitHub no markdown renderer will understand how to make # 13393 usable.

File: docs/changelogs/CHANGELOG-2.25.md [hunk 3]
```diff
@@ -4,8 +4,27 @@
...
 
-## [v2.25.3](https://github.com/kubermatic/kubermatic/releases/tag/v2.25.3)
+
+## v2.25.4
+
+**GitHub release: [v2.25.4](https://github.com/kubermatic/kubermatic/releases/tag/v2.25.4)**
+
+### Bugfixes
+
+- Fix [#13393](https://github.com/kubermatic/kubermatic/issues/13393) where externally deployed Velero CRDs are removed automatically from user user cluster ([#13396](https://github.com/kubermatic/kubermatic/pull/13396))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.25.md]: > user user cluster 

should probably be

> the user cluster

Comment (archups) [reply docs/changelogs/CHANGELOG-2.25.md]: nice catch

**Review by xrstf — APPROVED** [review]
/approve
/lgtm

**Pull request merged**